geometry_msgs/Pose left_arm
geometry_msgs/Pose right_arm